test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1498,6 +1498,9 @@ def forward(self, a):
 
         vec_amx = VecAMX()
         self._check_amx_counter(vec_amx)
+        if torch._C._cpu._is_amx_tile_supported():
+            # Only AMX ISA based micro-kernel is currently supported for da8w8
+            self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
```

Comment:
I think we should use the new counters add by you recently, right?

Reply:
No, because #147895 didn't get merged (timed out due to non-completion of some CI jobs)